packages/backend/src/accounting/tigerbeetle/utils.ts

```diff
@@ -1,21 +1,77 @@
 import { validateId } from '../../shared/utils'
+import { Transfer as TbTransfer, TransferFlags } from 'tigerbeetle-node'
+import { LedgerTransfer, LedgerTransferState, TransferType } from '../service'
+import { TigerBeetleTransferCode } from './service'
 
 export type AccountId = string | number | bigint
+export type AccountUserData128 = AccountId
 
-export function toTigerbeetleId(id: AccountId): bigint {
-  if (typeof id === 'number') {
-    return BigInt(id)
-  }
-  if (typeof id === 'bigint') {
-    return id
-  }
-  if (!validateId(id)) {
-    throw new Error('wrong format of id')
-  }
-
+export function toTigerBeetleId(id: AccountId): bigint {
+  if (typeof id === 'number') return BigInt(id)
+  else if (typeof id === 'bigint') return id
+  else if (!validateId(id)) throw new Error('wrong format of id')
   return uuidToBigInt(id)
 }
 
 export function uuidToBigInt(id: string): bigint {
   return BigInt(`0x${id.replace(/-/g, '')}`)
 }
+
+export function hexTextToBigInt(hex: string): bigint {
+  return BigInt(`0x${hex}`)
+}
+
+export function fromTigerBeetleId(bi: bigint): string {
```

Comment:
Why do we need this function? Why cant we keep the tiger beetle id as a big int? I'm guessing its to satisfy some check that we're returning a uuid (in graphql?) 

It looks like this is taking something that is not a UUID and making it look like a UUID. If its just to satsify some downstream validation that assumes uuid I think maybe just loosening that check makes more sense.

Reply:
> It looks like this is taking something that is not a UUID

Guess this understanding was wrong... I said that based off how TB is storing the ID but see we are doing `toTigerBeetleId(uuid())`.

Reply:
Yeah, for the settlement accounts we store the id as numeric (basically converts to a 1n), but for the liquidity accounts we make use of uuid()
Added test cases to `service.test.ts` to demonstrate this.